_includes/master/manifests/calico-typha.yaml

```diff
@@ -92,17 +92,13 @@ spec:
         livenessProbe:
           exec:
             command:
-            - calico-typha
-            - check
-            - liveness
+            - curl https://127.0.0.1:5473/liveness
```

Comment:
This shouldn't use curl here, it should use k8s' native HTTP readiness probe.  

See the example here: https://github.com/projectcalico/calico/blob/master/_includes/master/manifests/calico-node.yaml#L307